src/pages/welcome/WelcomePage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Redirect, useHistory } from 'react-router-dom';
+
+import './WelcomePage.css';
+import routes from 'constants/routes';
+import smilies from 'assets/smilies.svg';
+import Button from 'components/common/button/Button';
+
+const WelcomePage = () => {
+  const authenticated = useSelector(state => state.user.auth);
+  let history = useHistory();
+  const hangleClick = () => history.push('/dashboard');
+
+  if (!authenticated) {
+    return <Redirect to={routes.home} />;
+  }
+
+  return (
+    <div className="page-container">
+      <div className="column home-column-left">
+        <img src={smilies} alt="smilies"></img>
+        <span className="column__header column__header--no-transform column__header--no-bold">
```

Comment:
Not every piece of text should be a span. Appropriate hierarchy should be set, it gives the page structure and helps a lot with accessibility. https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals